tools/depends/target/Toolchain_binaddons.cmake.in

```diff
@@ -1,4 +1,9 @@
 set(OS "@platform_os@")
+set(CMAKE_SYSTEM_PROCESSOR @host_cpu@)
```

Comment:
This in case of Raspberry not correct enough, there it is only "arm" without something else, also on Android 32bit it is only "arm".

Reply:
See about Android here the standardized values of them https://github.com/Kitware/CMake/blob/master/Modules/Platform/Android-Determine.cmake#L243--L252 (on `..._PROC`)

Reply:
I'm torn, which would be better :smile:, if you go here it is fully OK https://cmake.org/cmake/help/latest/manual/cmake-toolchains.7.html.

Now looked over hundreds of CMakeLists.txt on the web, with the `set(CMAKE_SYSTEM_PROCESSOR ...)` many use it completely, but in the comparison I saw everywhere only for "arm" alone.

- The most make a `if (CMAKE_SYSTEM_PROCESSOR MATCHES "^arm")`
- Some are stupid and make a `if(CMAKE_SYSTEM_PROCESSOR STREQUAL "arm"`
- Some half stupid and make a `if(CMAKE_SYSTEM_PROCESSOR STREQUAL "arm" OR CMAKE_SYSTEM_PROCESSOR STREQUAL "armv7l")`

Then this path is actually OK, just a shame because of my work :cry: :laughing: :smile:.